### ISSUE TITLE
javax.imageio.IIOException: Can't create cache file!

### ISSUE THREAD

**Issue by PraiseArtoria** [Nettakrim/Signed-Paintings#39]
It's not uploading, only showing the link. (ver. 1.21.3)

**Comment by Nettakrim** [comment]
what link are you using specifically?

**Comment by PraiseArtoria** [comment]
> what link are you using specifically?

I click on 'upload on imgur' and use the imgur link it creates

**Comment by Nettakrim** [comment]
oh huh

**Comment by Nettakrim** [comment]
that could be bad

**Comment by PraiseArtoria** [comment]
It's not working also when i use the direct link from the picture

**Comment by Nettakrim** [comment]
okay hmm https://i.imgur.com/1Tx0Edb.png works for me on 1.21.3

**Comment by Nettakrim** [comment]
as does a newly uploaded link

**Comment by Nettakrim** [comment]
1) do you have any other mods that touch signs (eg stendhal, enhanced block entities)
2) send your latest.log, it should put some stuff in the console if it fails to load images

**Comment by PraiseArtoria** [comment]
1. I don't think so (EntityCulling or VanillaTweaks for commandblocks shouldn't do it?)
2. I'm sorry for the trouble but how can i do that?

**Comment by Nettakrim** [comment]
in your `.minecraft` folder (you can get to it by clicking on "open pack folder" in the resourcepacks screen and then going up a level in file explorer), go to the `logs` folder and look for `latest.log` (so the full directory will probably look like `...\AppData\Roaming\.minecraft\logs\latest.log`), then paste it into https://mclo.gs/, then send the link that generates

**Comment by PraiseArtoria** [comment]
https://mclo.gs/41ZjGIZ

**Comment by Nettakrim** [comment]
hmm `error downloading image: javax.imageio.IIOException: Can't create cache file!`, thats weird

**Comment by Nettakrim** [comment]
yeah i really dont know what to do about that, thats not directly an issue with the mod, thats an issue with java or your computer or something, mysterious

**Comment by PraiseArtoria** [comment]
It's fine, thank you tho :)

**Comment by PraiseArtoria** [comment]
I want to update. It's working for me now, using a different Mc Launcher fixed it!

**Comment by Nettakrim** [comment]
(i havent changed anything, this issue just isnt important if it is both not reproducible and also fixed itself)
